modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/client/ItThinClientComputeTest.java

```diff
@@ -493,6 +498,49 @@ void testExecuteColocatedOnUnknownUnitWithLatestVersionThrows() {
         assertEquals(INTERNAL_ERR, cause.code());
     }
 
+    @Test
+    void executeAsyncThrowsErrorWhenNodeIsNotFound() {
```

Comment:
It would be great to have a single test that runs against embedded and thin client APIs to ensure consistent behavior, like we do with `ItAbstractDataStreamerTest` or `ItSqlApiBaseTest`.